src/lucky_record/where.cr

```diff
@@ -2,14 +2,18 @@ module LuckyRecord::Where
   abstract class SqlClause
     getter :column, :value
 
-    def initialize(@column : Symbol | String, @value : String)
+    def initialize(@column : Symbol | String, @value : String | Array(String) | Array(Int32))
     end
 
     abstract def operator : String
     abstract def negated : SqlClause
 
     def prepare(prepared_statement_placeholder : String)
-      "#{column} #{operator} #{prepared_statement_placeholder}"
+      if ["= ANY", "!= ALL"].includes? operator
```

Comment:
If possible I guess we would like to not repeat those string literals here (and maybe get rid of the conditional?). I was going to suggest something like

```
operator = "ANY"

prepare = {
  "ANY" => ->(statement : String) { "(#{statement})" },
  "NOT ANY" => ->(statement : String) { "(#{statement})" }
  }.fetch(operator, ->(statement : String) { statement })


puts "#{prepare.call("$1")}"
puts "#{prepare.call("$3")}"
```

To replace the conditional by a lookup table but seeing it now its both more code and harder to understand on first sight haha. But still, removing the repeated operators could be a good idea?

Reply:
Yeah I agree removing repetition would be good if we end up with more conditions, but I think this is okay for now right?

Reply:
Sure, not a show stopper imho :) Alternatively we could make `operator` a function that accepts a string?

so e.g. for `Equal` this would get:
```crystal
def operator(placeholder : String)
    "= #{placeholder}"
end
```

and e.g. for  `In`
```crystaö
def operator(placeholder : String)
  "ANY (#{placeholder})"
end
```

Just throwing this out there :)

Reply:
Another option is to define a `prepare` method on `SqlClause` and then you can override it in `In` and `NotIn`. That would probably be simpler and would keep the customization in the class that needs it. LMK if you need me to expand on this or if it is not clear

Reply:
K added the `prepare` method to `In` and `NotIn`, that cleaned it up nicely